src/libs/utils.js

```diff
@@ -369,12 +369,30 @@ export const truncateInteger = integer => {
   return `${Math.floor(integer / 1000)}k`
 }
 
-// can this be done with usePollingEffect?
-export const poll = async (pollFn, pollTime, pollUntil) => {
-  let result = await pollFn()
-  while (!pollUntil(result)) {
+/**
+ Polls using a given function until the pollUntil function returns true.
+
+ @author srubenstein
+ @param {() => A} pollFn The function to poll using
```

Comment:
[It looks like](https://jsdoc.app/about-getting-started.html) we want a `-`, like so

```suggestion
 @param {() => A} pollFn - The function to poll using
```

Similarly for below.